### PR TITLE
tact1_modlist.json missing quotes

### PR DESCRIPTION
**Pull request by RaduM84**
Add a pair of missing quotes around the ACRE2 workshop ID in the tact1_modlist.json file.

### CODE REVIEW

File: novak/arma3/public/Tact1/tact1_modlist.json [hunk 1]
```diff
@@ -10,17 +10,18 @@
...
 		"@em":"333310405",
-		"@emr":"2034363662",
-		"@acre2":751965892		
+		"@emr":"2034363662",		
+		"@acre2":"751965892"
     },
     "server_mods":{
-		"@acre2":751965892
+	
+		"@acre2":"751965892"
```
Comment (ColdEvul) [inline novak/arma3/public/Tact1/tact1_modlist.json:18]:
```suggestion
		"@acre2":"751965892"
```

File: novak/arma3/public/Tact1/tact1_modlist.json [hunk 2]
```diff
@@ -10,17 +10,18 @@
...
     },
     "optional_mods":{
 		"@dui":"1638341685",
 		"@em":"333310405",
-		"@emr":"2034363662",
-		"@acre2":751965892		
+		"@emr":"2034363662",		
+		"@acre2":"751965892"
```
Comment (ColdEvul) [inline novak/arma3/public/Tact1/tact1_modlist.json:24]:
```suggestion
		"@acre2":"751965892"
```

File: novak/arma3/public/Tact1/tact1_modlist.json [hunk 3]
```diff
@@ -10,17 +10,18 @@
 		"@cup_vehicles":"541888371",
 		"@dui":"1638341685",
 		"@em":"333310405",
-		"@emr":"2034363662",
-		"@acre2":751965892		
+		"@emr":"2034363662",
```
Comment (ColdEvul) [inline novak/arma3/public/Tact1/tact1_modlist.json:13]:
```suggestion
		"@emr":"2034363662",
```

**Review by ColdEvul — CHANGES_REQUESTED** [review]
Minors but yet so important

File: novak/arma3/public/Tact1/tact1_modlist.json (hunk 2 above)
Comment (RaduM84) [reply novak/arma3/public/Tact1/tact1_modlist.json]: Deleted tab at line end in newest commit

File: novak/arma3/public/Tact1/tact1_modlist.json (hunk 1 above)
Comment (RaduM84) [reply novak/arma3/public/Tact1/tact1_modlist.json:18]: Deleted tab at line end in newest commit

**Review by ColdEvul — APPROVED** [review]
All fine for me

**Pull request merged**